### PR TITLE
chore: add missing migration changes

### PR DESCRIPTION
**Pull request by erik-slovak**

### CODE REVIEW

File: __tests__/Setup.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 **/
 
 import { configure } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
```
Comment (rostislav-simonik) [inline __tests__/Setup.ts:8]: we decided to get rid of the enzyme.

File: package.json [hunk 2]
```diff
@@ -49,39 +49,42 @@
...
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-redux-saga": "^1.3.2",
     "husky": "^8.0.1",
     "immutability-helper": "^3.1.1",
     "jest": "^29.1.2",
     "lint-staged": "^13.0.3",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```
Comment (rostislav-simonik) [inline package.json:77]: Why this is needed ?

File: package.json [hunk 3]
```diff
@@ -49,39 +49,42 @@
...
     "redux": "^4.2.0",
     "rimraf": "^3.0.2",
     "semantic-release": "^19.0.5",
     "semantic-release-slack-bot": "^3.5.3",
     "ts-jest": "^29.0.3",
     "typescript": "^4.8.4"
   },
+  "resolutions": {
+    "@types/react": "^17.0.0"
+  },
```
[the remark this answers is not in the document]
Comment (erik-slovak) [reply package.json:87]: Same thing as this: https://github.com/technology-studio/startup-react-peer/pull/16#discussion_r996950527

**Review by rostislav-simonik — APPROVED** [review]

**Pull request merged**